scripts/common/preflights.sh

```diff
@@ -287,6 +287,11 @@ selinux_enforced() {
 }
 
 function kotsadm_prerelease() {
+    if [ -z "$TESTGRID_ID" ]; then
```

Comment:
Not `-n`?

Reply:
whooops
I was going through this wondering "why are my changes not working"
and it was because this isn't addon code and so isn't in the PR test

I'll fix